tests/test_estimation.py

```diff
@@ -485,6 +485,13 @@ def test_weights_can_be_floats(self):
             warnings.simplefilter("always")
             kmf = KaplanMeierFitter().fit(T, E, weights=np.random.random(n))
             assert True
+    def test_float_weights():
```

Comment:
this test looks great! One thing need is to include `self` in the function here (because it's a class method on `TestKaplanMeierFitter`